modules/slack/resources/schema.yaml

```diff
@@ -0,0 +1,47 @@
+title: Schema for Jenkins Slack Notification plugin configuration
+type: object
+additionalProperties: false
+
+properties:
+  team_subdomain:
+    type: string
+    description: |
+      Your team's Slack subdomain. If you sign in to slack at https://example.slack.com/, your subdomain is 'example'.
+  token:
+    type: string
+    description: |
+      The integration token to be used to send notifications to Slack.
+      You can copy this from the settings page within Slack.
+  token_credentials_id:
```

Comment:
Well, I was about to write something about 'credentials' as the commonly used mostly plural form in this sense, but it is 'credential' in upstream plugin, so you're probably right here.